api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
@Emily-Jiang, I don't understand why a converter would want to throw an exception for null values? Empty value is a valid value and can always be converted to null with any converter.